metapool/igm.py

```diff
@@ -132,6 +132,33 @@ def pools(self, value):
 
         self._pools = value
 
+    def __str__(self):
+        out = (
+            f'Date of Sample Submission: {self.submission_date}\n'
+            f'Institute/Company Name: {self.institute}\n'
+            f'PI Name: {self.pi_name}\n'
+            f'PI Email: {self.pi_email}\n'
+            f'Contact Name: {self.contact_name}\n'
+            f'Contact Email: {self.contact_email}\n'
+            f'Project Number: {self.project_number}\n'
+            f'Task Number: {self.task_number}\n'
+            f'Platform: {self.platform}\n'
+            f'Run Type: {self.run_type}\n'
+            'Custom Primer? (Provide more info in comments box): '
+            f'{self.custom_primer}\n'
+            f'Total number of Samples: {self.number_of_samples}\n'
+            'Total number of Lanes OR Total Reads Required: '
+            f'{self.number_of_lanes}\n\n'
+            'Sample Name\tPool Name\tLibrary Size (bp)\tLibrary Prep Method\n'
+        )
+
+        if self.pools:
+            pools = '\n'.join(f'{p}\t{p}\t500\tKHP'for p in self.pools)
```

Comment:
What is `500`, and is it possible the pool would not be KHP?

Reply:
Good questions. 500 is "Library Size (bp)". For a bit more context, since our lab doesn't use IGM's demultiplexing infrastructure (which is just basespace), the contents of this table are not expected to be accurate. If you look at IGM's template [here](http://igm.ucsd.edu/genomics/docs/YYYY_MM_DD_PI_Sequencing_Runs_Manifest_2021.xlsx), you'll see that these values are entered in a table where indices, concentrations, and other fields would be expected. I assume it is possible for that value to not be KHP, but I'll need to check with MacKenzie. I'll get back to you when I hear back.

Reply:
I heard back from MacKenzie. She expects that these values will always be the same for metagenomics and don't foresee needing to change them.